src/OpenTelemetry.Api/Internal/ExceptionExtensions.cs

```diff
@@ -0,0 +1,40 @@
+﻿// <copyright file="ExceptionExtensions.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System;
+using System.Globalization;
+using System.Threading;
+
+namespace OpenTelemetry.Internal
+{
+    internal static class ExceptionExtensions
+    {
+        public static string ToInvariantString(this Exception exception)
```

Comment:
Haven't noticed this going on before. Anyone know why we're concerned about culture on the exceptions and not just doing a straight `.ToString()`?

Reply:
Probably to have a "normalized" way. But, not sure.

Reply:
Question is mainly for my own curiosity. I've done `Exception.ToString` a million times never considered that there is a culture issue with that 🤣

Reply:
This is for apps/services running in the customers environment where we have no control over the locale/culture settings.
The SDK has an intention to normalize the exception message (to English) so it can be easily aggregated/mined from the backend.

In a nutshell:
1. If you want the exception string for UI, use `ToString`.
2. If you want to collect exception string from different places and consume them in a central storage, use `ToInvariantString`.

Reply:
Looks like we want to put a comment here.

Reply:
I will use this (removed unintentionally) after the clean-up
```
/// <summary>	
        /// Returns a culture-independent string representation of the given <paramref name="exception"/> object,	
        /// appropriate for diagnostics tracing.	
        /// </summary>
```